src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -4207,8 +4215,25 @@ public void clearSnapshot(String tag, String... keyspaceNames)
             }
         }
 
+        Object olderThan = options.get("older_than");
```

Comment:
I was wondering if we need to support both options in the server or just `clearOlderThanTimestamp`, since we can easily construct that in nodetool from `olderThan` and not need to parse it in the server. On the other hand, if people access via JMX directly they might lose this functionality. WDYT?

Reply:
Imagine there is unix timestamp 1000 at client but time is different on server, like 1050. So when I specify 3h on the command line, I would take the current time on the command line (1000) and subtract 3h from it, for example, just to have an example, it would be timestamp 800.

But timestamp 800 is not 3 hours back from the timestamp 1050 on the server.

Please correct me if my reasoning is wrong here.